### PR TITLE
Add static code analyzes to CONTRIBUTING

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
Add static code analyzes to CONTRIBUTING

## :bulb: Motivation and Context
So people are able to replicate the CI steps too

### CODE REVIEW

**Comment by ueman** [comment]
You could mention that one can enable auto formatting in Android Studio and VS Code like this: https://flutter.dev/docs/development/tools/formatting

**Comment by marandaneto** [comment]
> You could mention that one can enable auto formatting in Android Studio and VS Code like this: https://flutter.dev/docs/development/tools/formatting

it's actually enabled by default on VSCode, the linter only actually checks if they are already formatted, the script runs a dryRun mode only, but indeed it'd be a nice addition if, for some reason, they have it disabled.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,3 +24,12 @@ The Flutter SDK has our Native SDKs embedded, if you wish to learn more about th
 * Android: Android SDK (`sentry-java`) with NDK (`sentry-native`): The example project includes C++.
 * iOS: Cocoa SDK (`sentry-cocoa`), you'll need a Mac with xcode installed.
 * Web: No additional dependencies.
+
+### Static Code Analysis, Tests, Formatting, Pub Score and Dry publish
+
+* Dart code
```
Comment (marandaneto) [inline CONTRIBUTING.md]:
```suggestion
* Dar/Flutter
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,3 +24,12 @@ The Flutter SDK has our Native SDKs embedded, if you wish to learn more about th
 * Android: Android SDK (`sentry-java`) with NDK (`sentry-native`): The example project includes C++.
 * iOS: Cocoa SDK (`sentry-cocoa`), you'll need a Mac with xcode installed.
 * Web: No additional dependencies.
+
+### Static Code Analysis, Tests, Formatting, Pub Score and Dry publish
+
+* Dar/Flutter
```
Comment (marandaneto) [inline CONTRIBUTING.md]:
```suggestion
* Dart/Flutter
```

**Pull request merged**